src/utility/PricingStrategy.js

```diff
@@ -0,0 +1,96 @@
+export class PricingStrategy {
+  constructor(pricingJSON) {
+    let pricingData = JSON.parse(pricingJSON);
```

Comment:
Looks like pricingData is not being mutated. Could be converted to `const`.

Reply:
Done